### PR TITLE
[CloudPC]Add enableMemAutoEnroll property

### PR DESCRIPTION
**Pull request by Litersam**

### CODE REVIEW

**Comment by AshleyYangSZ** [comment]
Please create a roadmap worktem and then attach the workitem number in this PR. Here is an example: https://microsoftgraph.visualstudio.com/Roadmap/_workitems/edit/14460/

If you don't have access, please do this firstly:

Request access via [Improved ADO Access - 1ES Wikirequest to join](https://www.1eswiki.com/wiki/Improved_ADO_Access) .
Join the Microsoft Graph Roadmap Users ([[[1]](mailto:graphroadmap@microsoft.com%7Cgraphroadmap@microsoft.com)]) security group on [idweb](https://idwebelements/GroupManagement.aspx?Group=graphroadmap&Operation=join) .

**Review by AshleyYangSZ — APPROVED** [review]

File: api-reference/beta/resources/cloudpcorganizationsettings.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ Represents the Cloud PC organization settings for a tenant. A tenant has only on
 |Property|Type|Description|
 |:---|:---|:---|
 |id|String|The ID of the organization settings.|
+|enableMEMAutoEnroll|Boolean|Specifies whether new Cloud PCs will be automatically enrolled in Microsoft Endpoint Manager(MEM). The default value is `false`.|
```
Comment (RuiHou105) [inline api-reference/beta/resources/cloudpcorganizationsettings.md:28]: One convention for graph public doc is: we'd better sort the properties alphabetically. So here in this table, please place the new property before` id`. Thanks.

**Review by RuiHou105 — CHANGES_REQUESTED** [review]
Left a minor comment. Please help resolve. Thanks.

**Review by RuiHou105 — APPROVED** [review]

**Comment by isvargasmsft** [comment]
Hi @RuiHou105, @AshleyYangSZ what is the status of this PR? Is it ready for review? I noticed you removed the "ready for content review" label.

**Comment by AshleyYangSZ** [comment]
> Hi @RuiHou105, @AshleyYangSZ what is the status of this PR? Is it ready for review? I noticed you removed the "ready for content review" label.

Hi @isvargasmsft this change doesn't go alive now, it's still bake in our testing env. Not sure shall we start review now or after the schema reach prod?

**Comment by isvargasmsft** [comment]
> > Hi @RuiHou105, @AshleyYangSZ what is the status of this PR? Is it ready for review? I noticed you removed the "ready for content review" label.
> 
> Hi @isvargasmsft this change doesn't go alive now, it's still bake in our testing env. Not sure shall we start review now or after the schema reach prod?

@AshleyYangSZ we can start once you have an schema PR approved :)

**Review by Guoan-Tang — CHANGES_REQUESTED** [review]
cloudpcorganizationsettings-get.md should be update too

**Comment by Litersam** [comment]
Already updated.

**Comment by Litersam** [comment]
> cloudpcorganizationsettings-get.md should be update too

Already updated.

**Review by Guoan-Tang — APPROVED** [review]

**Comment by Litersam** [comment]
roadmap work item:
14794 [DEVICES AND APPS][CLOUD PC] Add auto enroll MEM property to organizationSettings
https://microsoftgraph.visualstudio.com/Roadmap/_workitems/edit/14794

**Review by isvargasmsft — APPROVED** [review]

**Comment by RuiHou105** [comment]
Hi @Lauragra , this PR is ready for final review and the graph schema already went alive. So please help review/ approve it when you got time. Thanks!

**Pull request merged**